backend-api/index.js

```diff
@@ -29,6 +30,20 @@ app.get("/customers",async (req, res)=>{
     }
 })
 
+app.get("/games",async (req, res)=>{
+    let connection
+    try {
+        connection = await pool.getConnection()
+        const rows = await connection.query("SELECT machineID, name FROM machines")
```

Comment:
The naming convention assumes id for primary key. Do not use other names here

Reply:
The naming convention for this project is set so, that the descriptor for what the id is for is written in the name of the key. So instead of having just "Id" for everything, we will have Ids with their descriptors. like "machineID". this prevents confusion among the code when writing what handles the different IDs. If this is not acceptable, is there an alternative way to solve the problem of confusion among code? Just remembering is sometimes not an option for some people. Why is this not an accepted naming convention anyway? (A response of "Just is/isn't" does not explain why.) Different ids are used as foreign keys inside of other tables aswell, this will help keep things legible.